db/seeding.js

```diff
@@ -0,0 +1,34 @@
+const mongoose = require('mongoose');
+const Suggestion = require('./index.js');
+const faker = require('faker');
+
+for (let i = 1; i < 101; i++) {
+  let suggestions = [];
+  for (let j = 0; j < 10; j++) {
+    suggestions.push(faker.random.number(100));
```

Comment:
What is the suggestions array for?

Reply:
It stores the 10 suggested restaurants for each restaurant page.

Reply:
That's cool - thanks for clearing it up!